hawkbit-simple-ui/src/main/java/org/eclipse/hawkbit/ui/simple/view/TargetView.java

```diff
@@ -286,7 +342,8 @@ private RegisterDialog(final HawkbitMgmtClient hawkbitClient) {
             final Button register = Utils.tooltip(new Button("Register"), "Register (Enter)");
             type = new Select<>(
                     "Type",
-                    e -> {},
+                    e -> {
+                    },
                     hawkbitClient.getTargetTypeRestApi()
                             .getTargetTypes(0, 30, Constants.NAME_ASC, null)
```

Comment:
@strailov see, same here too. I guess this will need a separate PR to handle thses cases